pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[tool.pytest.ini_options]
+pythonpath = '.'
+
+[tool.ruff]
+select = ["E", "F", "B", "UP", "C4", "SIM"]
+ignore = ["E501"]
+target-version = "py39"
```

Comment:
Debian bullseye's default python3 is 3.9. Set this as the minimum version for now in case we want to run custom code on a coprocessor.